irr/include/matrix4.h

```diff
@@ -242,17 +242,11 @@ class CMatrix4
 	//! Translate a vector by the inverse of the translation part of this matrix.
 	void inverseTranslateVect(vector3df &vect) const;
 
-	//! Rotate a vector by the inverse of the rotation part of this matrix.
-	void inverseRotateVect(vector3df &vect) const;
+	//! Scale a vector, then rotate by the inverse of the rotation part of this matrix.
+	void scaleThenInvRotVect(vector3df &vect) const;
```

Comment:
You could instead write down that it has to be an orthonormal matrix.

Reply:
This would defeat the purpose of this PR IMHO.

Reply:
> You could instead write down that it has to be an orthonormal matrix.

The problem is that in general it isn't. Hence the rename, to remind callers to renormalize if necessary. It's obvious that for an orthonormal matrix the scaling part does nothing, though if you want I can document that.